apps/spruce/src/hooks/useScrollToAnchor/useScrollToAnchor.test.tsx

```diff
@@ -25,7 +25,7 @@ describe("useScrollToAnchor", () => {
     );
 
     renderHook(() => useScrollToAnchor(), { wrapper });
-    vi.advanceTimersByTime(500);
+    vi.advanceTimersByTime(anchorScrollTime);
```

Comment:
```suggestion
    vi.runOnlyPendingTimers();
```

Will do the same thing and not rely on `anchorScrollTime`